Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/ComponentBroadcasterService.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// Types of state changes that may occur for an <see cref="IComponentBroadcaster"/>
+    /// </summary>
+    public enum ComponentBroadcasterChangeType : byte
```

Comment:
>public enum ComponentBroadcasterChangeType : byte [](start = 4, length = 49)

There are several different classes in this file, not all of them related (I think this was just a convenient dumping ground in the past). It would be nice to try to stick to one class per file going forward.

Reply:
I'm going to address this in a follow up review to unblock other work by Andrei

---
In reply to: [284497284](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4386#discussion_r284497284) [](ancestors = 284497284)